--- origsrc/lxqt-panel-0.11.0/cmake/BuildPlugin.cmake	2016-09-23 17:45:47.000000000 -0500
+++ src/lxqt-panel-0.11.0/cmake/BuildPlugin.cmake	2016-11-06 21:27:31.575263100 -0600
@@ -58,8 +58,10 @@ MACRO (BUILD_LXQT_PLUGIN NAME)
     if (${IS_STATIC} EQUAL -1) # not static
         add_library(${NAME} MODULE ${SRC}) # build dynamically loadable modules
         install(TARGETS ${NAME} DESTINATION ${PLUGIN_DIR}) # install the *.so file
+        target_link_libraries(${NAME} lxqt-panel)
     else() # static
         add_library(${NAME} STATIC ${SRC}) # build statically linked lib
+        add_definitions(-DCOMPILE_LXQT_PANEL)
     endif()
     target_link_libraries(${NAME} ${QTX_LIBRARIES} lxqt ${LIBRARIES} KF5::WindowSystem)
 
--- origsrc/lxqt-panel-0.11.0/panel/CMakeLists.txt	2016-09-23 17:45:47.000000000 -0500
+++ src/lxqt-panel-0.11.0/panel/CMakeLists.txt	2016-11-06 21:06:10.433854600 -0600
@@ -106,6 +106,8 @@ add_executable(${PROJECT}
     ${UI}
 )
 
+set_target_properties(${PROJECT} PROPERTIES ENABLE_EXPORTS ON)
+
 target_link_libraries(${PROJECT}
     ${LIBRARIES}
     ${QTX_LIBRARIES}
--- origsrc/lxqt-panel-0.11.0/plugin-volume/CMakeLists.txt	2016-09-23 17:45:47.000000000 -0500
+++ src/lxqt-panel-0.11.0/plugin-volume/CMakeLists.txt	2016-11-06 21:40:28.588562700 -0600
@@ -10,7 +10,6 @@ set(HEADERS
 )
 
 set(SOURCES
-    ${PROJECT_SOURCE_DIR}/panel/lxqtpanelpluginconfigdialog.cpp
     lxqtvolume.cpp
     volumebutton.cpp
     volumepopup.cpp
