--- origsrc/plugin-dom/CMakeLists.txt	2014-04-30 06:24:36.000000000 -0500
+++ src/plugin-dom/CMakeLists.txt	2014-06-08 18:49:36.110555400 -0500
@@ -28,6 +28,8 @@ set(RESOURCES
     resources.qrc
 )
 
+set(LIBRARIES ${QTXDG_LIBRARIES})
+
 #*******************************************
 include ("../BuildPlugin.cmake")
 BUILD_LXQT_PLUGIN(${PLUGIN})
--- origsrc/plugin-kbindicator/lxqtkbindicatorconfiguration.h	2014-04-30 06:24:36.000000000 -0500
+++ src/plugin-kbindicator/lxqtkbindicatorconfiguration.h	2014-06-08 19:03:23.090855900 -0500
@@ -50,12 +50,6 @@ public:
 public slots:
     void saveSettings();
 
-    void selectColour(const QString &);
-    void on_typeCOB_currentIndexChanged(int);
-    void on_sourceCOB_currentIndexChanged(int);
-    void on_maximumHS_valueChanged(int);
-    void on_fontB_clicked();
-
 
 private:
     Ui::LxQtKbIndicatorConfiguration *ui;
--- origsrc/plugin-worldclock/lxqtworldclock.h	2014-04-30 06:24:36.000000000 -0500
+++ src/plugin-worldclock/lxqtworldclock.h	2014-06-08 18:59:18.877887800 -0500
@@ -66,7 +66,6 @@ public:
     QDialog *configureDialog();
 
 private slots:
-    void synchroTimeout();
     void timeout();
     void wheelScrolled(int);
 
