--- origsrc/lxqt-panel-0.8.0/plugin-kbindicator/lxqtkbindicatorconfiguration.h	2014-10-13 10:27:26.000000000 -0500
+++ src/lxqt-panel-0.8.0/plugin-kbindicator/lxqtkbindicatorconfiguration.h	2014-11-25 18:50:05.348879000 -0600
@@ -50,13 +50,6 @@ public:
 public slots:
     void saveSettings();
 
-    void selectColour(const QString &);
-    void on_typeCOB_currentIndexChanged(int);
-    void on_sourceCOB_currentIndexChanged(int);
-    void on_maximumHS_valueChanged(int);
-    void on_fontB_clicked();
-
-
 private:
     Ui::LxQtKbIndicatorConfiguration *ui;
     QSettings *mSettings;
--- origsrc/lxqt-panel-0.8.0/plugin-worldclock/lxqtworldclock.h	2014-10-13 10:27:26.000000000 -0500
+++ src/lxqt-panel-0.8.0/plugin-worldclock/lxqtworldclock.h	2014-11-25 18:50:53.403481200 -0600
@@ -74,7 +74,6 @@ public:
     QDialog *configureDialog();
 
 private slots:
-    void synchroTimeout();
     void timeout();
     void wheelScrolled(int);
 
