--- origsrc/lxqt-panel-0.9.0/panel/lxqtpanelpluginconfigdialog.h	2015-02-08 03:31:27.000000000 -0600
+++ src/lxqt-panel-0.9.0/panel/lxqtpanelpluginconfigdialog.h	2015-04-28 20:52:03.858188000 -0500
@@ -36,7 +36,11 @@
 
 class QComboBox;
 
+#ifdef LXQT_PLUGIN_COMPILATION
+class LxQtPanelPluginConfigDialog : public QDialog
+#else
 class LXQT_PANEL_API LxQtPanelPluginConfigDialog : public QDialog
+#endif
 {
     Q_OBJECT
 public:
--- origsrc/lxqt-panel-0.9.0/plugin-volume/CMakeLists.txt	2015-02-08 03:31:27.000000000 -0600
+++ src/lxqt-panel-0.9.0/plugin-volume/CMakeLists.txt	2015-04-28 20:51:02.180355900 -0500
@@ -9,6 +9,8 @@ set(HEADERS
     audioengine.h
 )
 
+add_definitions(-DLXQT_PLUGIN_COMPILATION)
+
 set(SOURCES
     ${PROJECT_SOURCE_DIR}/panel/lxqtpanelpluginconfigdialog.cpp
     lxqtvolume.cpp
