--- origsrc/lxqt-panel-0.9.0/plugin-showdesktop/CMakeLists.txt	2015-04-28 20:52:49.144438600 -0500
+++ src/lxqt-panel-0.9.0/plugin-showdesktop/CMakeLists.txt	2015-04-28 20:48:33.986037600 -0500
@@ -1,5 +1,8 @@
 set(PLUGIN "showdesktop")
 
+include(FindPkgConfig)
+pkg_check_modules(XCB REQUIRED xcb)
+
 set(HEADERS
     showdesktop.h
 )
@@ -16,6 +19,7 @@ set(LIBRARIES
     ${LIBRARIES}
     ${LXQT_GLOBALKEYS_LIBRARIES}
     ${QTXDG_LIBRARIES}
+    ${XCB_LIBRARIES}
 )
 
 BUILD_LXQT_PLUGIN(${PLUGIN})
